### PR TITLE
initial implementation

### PR DESCRIPTION
**Pull request by benghaziboy**
@mparent61 

This will be imported into the respective reading_lists to add the appropriate query.

### CODE REVIEW

File: bulbs/special_coverage/search.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    :param query2: Secondard search that will return at the 1st index of iteration.
+    :type  query2: Any iterable object; intended for djes.search.LazySearch and django.Querysets.
+    """
+    index = 0
+    offset = 0
+    while True:
+        result = None
+        if index == 1:
+            try:
+                result = query2[0]
```
Comment (mparent61) [inline bulbs/special_coverage/search.py]: Do queries support a next() iterface? Then you wouldn't need to track offset
`result = next(query2)`
(We may have been down this road before -- perhaps queries behave beset when indexed via []? )

Comment (benghaziboy) [inline bulbs/special_coverage/search.py]: currently they do not.

**Comment by mparent61** [comment]
Otherwise LGTM 
I think this is the most flexible approach. Only immediate downside is the extra code to check for IndexErrors and such -- maybe that could be streamlined by some wrapper if we continue this approach?

**Comment by benghaziboy** [comment]
@mparent61 updated this to include the `next` calls from the most recent djes updates

**Comment by mparent61** [comment]
Much cleaner with next().
As I think you mentioned in the djes update, will the next() call make repeated ES backend queries on subsequent calls against the same query? Or will caching ensure ES is only hit once?

**Comment by benghaziboy** [comment]
@mparent is this good to go?

**Comment by MichaelButkovic** [comment]
:+1:

**Comment by benghaziboy** [comment]
blessings

**Pull request merged**